src/web/mjs/engine/Connectors.mjs

```diff
@@ -63,7 +63,30 @@ export default class Connectors {
             console.warn(`Failed to load connector`, error);
         }
     }
-    
+
+    async registerfav(files) {
+        try {
+            for(let file of files) {
+                try {
+                    let module = await import(file);
+                    let connector = new module.default();
+                    if(this._favlist.find(c => c.id === connector.id)) {
+                        console.warn(`The connector "${connector.label}" with ID "${connector.id}" is already registered`);
+                    } else {
+                        this._favlist.push(connector);
+                    }
+                } catch(error) {
+                    console.warn(`Failed to load connector "${file}"`, error);
+                }
+            }
+            this._favlist.sort( ( a, b ) => {
+                return ( a.label.toLowerCase() < b.label.toLowerCase() ? -1 : 1 );
+            } );
+        } catch(error) {
+            console.warn(`Failed to load connector`, error);
+        }
+    }
+
```

Comment:
only part I'm not happy with, since original `register(files)` uses `this._list` (which can't be rewritten) I've had to create a new register function.

Reply:
Method is not needed if you accept the suggested change based on `Storage.mjs`